<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlConnection</class>
 <widget class="QDialog" name="SqlConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>SQL data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1" colspan="3">
       <widget class="QTableView" name="tableView">
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>261</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>261</width>
          <height>291</height>
         </size>
        </property>
        <property name="title">
         <string>SQL connection</string>
        </property>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>25</y>
           <width>241</width>
           <height>102</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Db table name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="dbEntry">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Open database</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>250</y>
           <width>160</width>
           <height>26</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>140</y>
           <width>200</width>
           <height>91</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ID">
              <property name="maximumSize">
               <size>
                <width>110</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>ID of measurement</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Normal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QCommandLinkButton" name="commandLinkButton">
            <property name="maximumSize">
             <size>
              <width>201</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>Show database</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionFetchDb">
   <property name="text">
    <string>fetchDb</string>
   </property>
  </action>
  <action name="actionInsertRow">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>insert row</string>
   </property>
   <property name="statusTip">
    <string>Insert a new row</string>
   </property>
  </action>
  <action name="actionDeleteRow">
   <property name="text">
    <string>delete row</string>
   </property>
   <property name="statusTip">
    <string>Delete a row</string>
   </property>
  </action>
  <action name="actionShowDbTable">
   <property name="text">
    <string>show Db table</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SqlConnection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SqlConnection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLinkButton</sender>
   <signal>clicked()</signal>
   <receiver>actionFetchDb</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
